modules/styleguide/assets/js/frontend/components/areas/area-title.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const AreaTitle = styled.h2`
+	color: #515962;
```

Comment:
Please use a variable (try to use an existing one)

Reply:
Fixed